kebechet/base/openshift-templates/kebechet-run-url.yaml

```diff
@@ -0,0 +1,108 @@
+---
+# This template uses the kebechet-run-url argo workflow to run kebechet on a particular slug name.
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: kebechet-run-url
```

Comment:
In the stage and prod kustomize file, we would need to update that this file has to be deployed to infra-stage.
update in a new line below: https://github.com/thoth-station/thoth-application/blob/d3257ddddfd599b747111e25fceb2d29052a83bc/kebechet/overlays/stage/kustomization.yaml#L29
with:
```
  - path: put-into-infra-namespace.yaml
    target:
      group: template.openshift.io
      version: v1
      kind: Template
      name: kebechet-run-url
```

Reply:
Thanks I have added it in the customize files now :+1: